src/tests/services/edge-application-cache-settings-services/load-cache-settings-service.test.js

```diff
@@ -1,5 +1,5 @@
 import { AxiosHttpClientAdapter } from '@/services/axios/AxiosHttpClientAdapter'
-import { loadCacheSettingsService } from '@/services/edge-application-cache-settings-services'
+import { loadCacheSettingsService } from '@/services/edge-application-edge-cache-services'
```

Comment:
loadCacheSettingsService -> loadEdgeCacheService

Reply:
Tinha esquecido de subir. ja to subindo